assets/_scss/elements/_list.scss

```diff
@@ -21,7 +21,7 @@ article {
     }
```

Comment:
Why is this under an article rule? What happens if I want this list style when I'm not using an article? What about users that aren't really using article tags because they don't know much about html5 elements? How it is going to be documented that you need to nest it in an article tag?